### PR TITLE
Merge and update release note for Deploy Windows

### PR DESCRIPTION
**Pull request by YuzuruUeda**

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -10,6 +10,13 @@ Release Date: 2018-03-05
 
 New Features
 ~~~~~~~~~~~~
+* The migration process (scan, generate and publish) can now be automated by using a simple and intuitive work-flow in the user interface. The user simply creates a new migration through the interface, launches it from the server to migrate and follows the overall status.
```
Comment (emuus) [inline changelog.rst]: workflow doesn't need hyphen

File: changelog.rst [hunk 2]
```diff
@@ -10,6 +10,13 @@ Release Date: 2018-03-05
...
+
+	.. note::
+
+		* This new feature is available under the "Migrations" tab. This tab already existed. This previous tab has been renamed "Scans".
```
Comment (emuus) [inline changelog.rst]: This tab already existed. This previous tab has been renamed "Scans". -> Is not terribly clear.
I would suggest : 
This new feature is available under the "Migrations" tab. The manual migration workflow using Scan, generate and publish is now available through the new "Scans" tab.

**Comment by YuzuruUeda** [comment]
Hello Elaine-san,
I talked with Jeff-san, PO of SW team, and agreed to take the following steps.
1) Merge my change
2) Then, SW team will update Release Note based on your comment

Is it OK for you?

**Comment by emuus** [comment]
yes ok for me

**Comment by YuzuruUeda** [comment]
Hello Elaine-san,
Thank you for your understanding.
Could you "accept" my change on this PR?

**Pull request merged**